src/js/index.js

```diff
@@ -396,6 +397,47 @@ function handleBoardDragEnd(e) {
   App Functions
 */
 
+function toggleTheme() {
+  if (theme === "light") {
+    document.body.className = theme = "dark";
+    setLocalStorageItem("theme", "dark");
```

Comment:
Would you mind if we prefixed the local storage key with _manifest__? Helps keep in line with the memos key.